config-templates/ibm/ibm-csi-block/1.4.0/README.md

```diff
@@ -0,0 +1,101 @@
+# IBM block storage Container Storage Interface driver
+
+IBM block storage CSI driver is based on an open-source IBM project, included as a part of IBM Storage orchestration for containers. IBM Storage orchestration for containers enables enterprises to implement a modern container-driven hybrid multicloud environment that can reduce IT costs and enhance business agility, while continuing to derive value from existing systems.
+
+For full release notes, compatiblity, installation, and user information, see the IBM block storage CSI driver documentation: https://www.ibm.com/support/knowledgecenter/SSRQ8T_1.4.0/
+
+## Prerequisites
+
+For full prerequisite instructions, see the following documentation, referred to in the introductory paragraph. 
+**Installation** > **Compatibility and requirements**.
+
+**Important:** Be sure to complete all prerequisite and installation steps before assigning hosts to your location. Do not create a Kubernetes cluster. This is done through Satellite.
+
+## IBM block storage CSI driver parameters & how to retrieve them
+
+
+
+Retrieve all parameters required by this template.
+
+```
+
+ibmcloud sat storage template get --name ibm-csi-block --version 1.4.0
+
+```

```

Comment:
```suggestion
```
ibmcloud sat storage template get --name ibm-csi-block --version 1.4.0
```

```

Reply:
what is the fix?

Reply:
I changed, hope it what you meant

Reply:
removing the extra linebreaks on L19 & 21

Reply:
ok, done